src/App.js

```diff
@@ -48,14 +55,26 @@ function App() {
       ];
     });
   };
+  const keydownhandler = (e) => {
+    if (showInput) {
+      if (e.key === "Escape") {
+        // console.log("esc clicked");
+        setShowInput(false);
+      }
+    }
+  };
   return (
-    <div className={css.container}>
+    <div tabIndex={"0"} className={css['layout-container']} onKeyDown={keydownhandler}>
+    <main className={css.container}>
       <TodoCard addedTask={tasksUpdated} setToDos={setTasksUpdated} />
-      {showInput && (
+      {showInput ? (
         <NewTask gotEscReq={escReqHandler} getData={getDataHandler} />
-      )}
-      {!showInput && <AddTaskButton onButtonClicked={buttonClickedHandler} />}
-    </div>
+        ) : (
+          <AddTaskButton onButtonClicked={buttonClickedHandler} />
+          )}
+    </main>
+          </div>
+    
```

Comment:
Format above few lines of code with proper spacing.
Remove commented code from all files.
Solve #4